### ISSUE TITLE
Unable to automatically download/install artifact 'oneTBB' from sources

### ISSUE THREAD

**Issue by celise88** [JuliaLang/julia#54368]
I have an employee who's attempting to install packages MCMC and Turing in Julia, both of which have oneTBB as a dependency. He is using Julia version 1.10.3 and getting the error message below. Other packages install without any problems. 

LoadError: Unable to automatically download/install artifact 'oneTBB' from sources listed in 'C:\Users\a0824021\.julia\packages\oneTBB_jll\QvAaw\Artifacts.toml'.
Sources attempted:
- https://pkg.julialang.org/artifact/a80da89d0fb528913efffff81a0cf1e2ab988cb2
    Error: RequestError: HTTP/1.1 403 Forbidden while requesting https://pkg.julialang.org/artifact/a80da89d0fb528913efffff81a0cf1e2ab988cb2
- https://github.com/JuliaBinaryWrappers/oneTBB_jll.jl/releases/download/oneTBB-v2021.12.0+0/oneTBB.v2021.12.0.x86_64-w64-mingw32-cxx11.tar.gz
    Error: RequestError: HTTP/1.1 403 Forbidden while requesting https://github.com/JuliaBinaryWrappers/oneTBB_jll.jl/releases/download/oneTBB-v2021.12.0+0/oneTBB.v2021.12.0.x86_64-w64-mingw32-cxx11.tar.gz

I'm not having any issues with Julia 1.9, but I'm on a MacOS and he is on Windows. 

Any help would be greatly appreciated.

**Comment by Keno** [comment]
The artifact definitely exists in both location. Additionally, it would be unlikely that they would both return 403 on a misconfiguration. Most likely they have some sort of firewall or other network filter installed that is blocking these requests. Did they try downloading manually with the browser? That workflow often has a firewall exception hook to allow this.

**Comment by celise88** [comment]
I have not asked him to try manually downloading yet. Firewall was my first thought given the 403 - I just thought it was odd that he didn't have problems installing any other packages if that was the issue.

**Comment by fingolfin** [comment]
I don't think we can do much about this at this point without additional details. Since it's also been 10 months I am closing this now. But of course this can be reopened if necessary, or open a new issue...
